CONTRIBUTING.md

```diff
@@ -3,45 +3,55 @@
 We'd love to have you contribute to Loris. The first thing you should do
 before contributing is probably sign up for the [LORIS developers' mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev).
 
+Your next step before issuing a pull request is to review our [Coding Standards](./docs/CodingStandards.md). If you are doing front-end development you should also check out our [React guidelines](./LORIS_react.README.md).
+
+You can also learn about our code review process by perusing our [Code Review Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) in the [LORIS Wiki](https://github.com/aces/Loris/wiki). These will be some of the factors we'll consider when reviewing your code.
+
 ## Development branches
 
-Pull requests should be based on and sent to the VERSIONNUMBER-dev branch of the version that you are targeting. The master branch is reserved for stable releases.
+You should base your pull requests on one of the following branches depending on the kind of change you are making:
+
+#### Bug Fixes
+   - Branch: `bugfix`
+   - Label: ![](https://via.placeholder.com/15/cc9966/000000?text=+) **[branch] bugfix**
```

Comment:
"https://via.placeholder.com/15/cc9966/000000?text=+"?

Reply:
I had added these to the wiki, thats the only way I found to put colourful boxes to match the label colours